src/components/forms/DateInputGroup/DateInputGroup.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export const DateInputGroup = (
+  props: React.HTMLAttributes<HTMLElement>
+): React.ReactElement => {
+  const { children, ...divAttributes } = props
+
+  return (
+    <div
+      className="usa-memorable-date"
```

Comment:
Since `divAttributes` can include a custom `className`, we should use `classnames` here to append `props.className` to `usa-memorable-date` so it doesn't get overwritten